chain-signatures/node/src/mesh/connection.rs

```diff
@@ -49,32 +49,37 @@ impl Pool {
                 continue;
             };
 
-            let Ok(resp) = self
-                .http
-                .get(url.clone())
-                .timeout(Duration::from_millis(200))
-                .send()
-                .await
-            else {
-                tracing::warn!(
-                    "Pool.ping resp err participant {:?} url {}",
-                    participant,
-                    url
-                );
-                continue;
+            let work = self.http.get(url.clone()).send();
```

Comment:
Since we have issues with node stability, should we add this info to metrics? Each node can report the number of active participants. We can even add an alert on this.

Reply:
sounds good! I'll do it in another PR since this PR will already be big